### PR TITLE
Script to monitor for slow workers

### PR DESCRIPTION
**Pull request by markwallsgrove**
This script adds the ability to check for workers that are taking too long to process a Resque job.

### CODE REVIEW

**Comment by mattyjones** [comment]
@markwallsgrove  Can you please take a look at the Travis output.  Thanks!

**Comment by eheydrick** [comment]
Bump for rubocop issues.

**Comment by markwallsgrove** [comment]
@mattyjones @eheydrick checks are now passing.

File: bin/check-timeouts-resque.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright 2015 Mark Wallsgrove <mw@talis.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
```
Comment (eheydrick) [inline bin/check-timeouts-resque.rb]: This isn't needed since the plugin only support Ruby 1.9.3+

**Comment by eheydrick** [comment]
@markwallsgrove. Thanks for fixing the tests. I left one comment, can you take a look. Can you also add the new plugin to the README and add a CHANGELOG entry under unreleased. Once that's addressed I'll merge.

**Comment by markwallsgrove** [comment]
@eheydrick shall I squash the commits?

**Comment by markwallsgrove** [comment]
@eheydrick all done :)

**Comment by eheydrick** [comment]
Looks good, thanks!

**Pull request merged**